docs/tech-reference/importer.rst

```diff
@@ -13,8 +13,14 @@ Properties
  The URL for the upstream ostree repository to sync.
 
 ``branches``
- A list of branches from the upstream repo that should be pulled during a sync.
+ An array of branch names from the upstream repository that should be pulled
+ during a sync. If an empty array is provided, the list of branches will be
+ populated on the repository scratchpad, but no branches will be retrieved. The
+ default is to retrieve all branches.
 
 ``depth``
  The tree traversal depth. This determines how much history is pulled from the remote.
  A value of ``-1`` indicates infinite. The default is: ``0``.
+
+``gpg_keys``
+ A list of keys that can be used to validate signed branches.
```

Comment:
Good point. I think I copy-pasted this from another part of these docs, figuring some mention of it is better than none. I should probably clarify what exactly this should be. Maybe @jortel can add some insight.